src/importexport/guitarpro/internal/gtp/gp67dombuilder.cpp

```diff
@@ -276,8 +276,8 @@ std::vector<GPMasterTracks::Automation> GP67DomBuilder::readTempoMap(XmlDomNode*
                 tempo.type = GPMasterTracks::Automation::Type::tempo;
                 String str = currentAutomation.firstChildElement("Value").toElement().text();
                 StringList tempoValue = str.split(u' ');
-                tempo.value = tempoValue[0].toInt();
-                tempo.tempoUnit = tempoValue.size() > 1 ? tempoValue.at(1).toInt() : 0;
+                tempo.value = static_cast<int>(tempoValue[0].toDouble());
+                tempo.tempoUnit = tempoValue.size() > 1 ? static_cast<int>(tempoValue.at(1).toDouble()) : 0;
```

Comment:
Why is the `[]` operator used to access element 0, but the `at` method is used to access element 1?

Reply:
right, didn't change how it was, will fix